src/jbi/whiteboard_actions/default.py

```diff
@@ -215,7 +215,7 @@ def create_and_link_issue(
         jira_response = self.jira_client.create_or_update_issue_remote_links(
             issue_key=jira_key_in_response,
             link_url=bugzilla_url,
-            title="Bugzilla Ticket",
```

Comment:
Would it be redundant to include the context that it's a Bugzilla bug? `Bugzilla Bug XXXXXXX`

Reply:
Yeah, you're right